go-controller/pkg/cni/helper_linux.go

```diff
@@ -294,6 +284,9 @@ func ConfigureOVS(ctx context.Context, namespace, podName, hostIfaceName string,
 		"other_config:transient=true",
 	}
 
+	if len(ifInfo.VfNetdevName) != 0 {
+		ovsArgs = append(ovsArgs, "external_ids:vf-netdev-name=%s", ifInfo.VfNetdevName)
```

Comment:
Shall we add vf-representor-name as one of the port external ids and recover it on cmdDel?

Reply:
i am not sure it is required since it is always going to be the first 15 bytes of the Pod's sandbox ID.